compiler/luci-value-test/tester/src/EvalTester.cpp

```diff
@@ -39,9 +39,17 @@ void readDataFromFile(const std::string &filename, char *data, size_t data_size)
     throw std::runtime_error("Failed to read data from file \"" + filename + "\".\n");
 }
 
-void writeDataToFile(const std::string &filename, const char *data, size_t data_size)
+void writeDataToFile(const std::string &filename, const char *data, size_t data_size, bool append)
 {
-  std::ofstream fs(filename, std::ofstream::binary);
+  std::ofstream fs;
+  if (append)
+  {
+    fs = std::ofstream(filename, std::ofstream::app);
+  }
+  else
+  {
+    fs = std::ofstream(filename, std::ofstream::binary);
+  }
```

Comment:
(optional) It would be possible to reduce the code as below

```
void writeDataToFile(const std::string &filename, const char *data, size_t data_size, ios_base::openmode mode)
{
  std::ofstream fs = std::ofstream(filename, mode);
```

Reply:
Okay, I will fix it.